guidelines/style-guidelines.md

```diff
@@ -77,18 +81,17 @@ We use double quotation marks for:
 * direct speech.
 
 We use single quotation marks for:
-* technical terms (the first time it is used)
-* classification descriptors
+* technical terms, if deemed necessary (the first time it is used)
 * titles of documents or publications
 
 
 ## Links
 
 We:
 * link to relevant content on our website before we link to external websites 
-* never link headings
+* never use links in headings
 * write descriptive links that tell you what you’ll find when you follow them - not 'click here'
-* don't set links to open in a new tab or window, with exceptions to be decided case by case.
+* links inside editing systems should always open in new tabs to not disrupt a user's workflow. In most other circumstances we don't set links to open in a new tab or window.
```

Comment:
You now made this into two sentences, but these lists are not supposed to be of full sentences ;) It also does not follow from "we" :stuck_out_tongue: Maybe split into two points? We: open links inside [...] to avoid disrupting [...] / We: "don't set links [...] in most other circumstances" ?